README.md

```diff
@@ -1,5 +1,6 @@
 # OSPO Project Template
 
+![Release](https://img.shields.io/badge/OSS%20Template-0.3.1-brightgreen.svg)
```

Comment:
It's hard coded as green? do we care if they are X versions behind? Is that on them or us to keep up to date with the template if at all?

Reply:
As above, I think it's important that we not make this overly dynamic as it's meant to represent a fixed point in time. The green, for me, was to distinguish it from the colors used for the code of conduct, etc. Maybe changing it to blue to avoid the unwanted stoplight analogy would be better?

Reply:
Yeah, agree that non-stoplight color might help avoid that line of thinking. Unless we think there is value in wanting repos to stay up to date with the template changes and then signaling it being 'out of date' wouldn't be a bad thing?

Reply:
I _think_ I'd prefer to shy away from that. In my mind, this gives them a point of comparison and a list of difference for if/when they choose to ingest upstream changes. What I'd like to avoid is having this become something akin to Dependabot.

Reply:
We could always go with purple, cuz, y'know, branding and all 😛 

Example using the Wayfair primary hex code (`7f187f`):

![Version](https://img.shields.io/badge/version-1.0.0-7f187f.svg?cacheSeconds=2592000)

Raw URL: https://img.shields.io/badge/version-1.0.0-7f187f.svg?cacheSeconds=2592000